src/snowflake/snowpark/modin/pandas/base.py

```diff
@@ -1735,16 +1735,38 @@ def ewm(
             method=method,
         )
 
-    @base_not_implemented()
     def expanding(
         self, min_periods=1, axis=0, method="single"
     ):  # noqa: PR01, RT01, D200
         """
         Provide expanding window calculations.
         """
-        # TODO: SNOW-1119855: Modin upgrade - modin.pandas.base.BasePandasDataset
```

Comment:
This ticket is not done yet. Can we keep the todo?

Reply:
This TODO should be done even though the overall ticket may not be, the code below for expanding is a straight copy-paste from Modin